Marlin/src/feature/pause.cpp

```diff
@@ -252,94 +256,143 @@ void wait_for_filament_reload(const int8_t max_beep_count/*=0*/) {
 }
 
 void resume_print(const float &load_length/*=0*/, const float &initial_extrude_length/*=0*/, const int8_t max_beep_count/*=0*/) {
-  bool nozzle_timed_out = false;
-
   if (!move_away_flag) return;
-
+  bool nozzle_timed_out = false;
+  bool swap_spool_disabled = true;
+  int16_t fansp=fanSpeeds[PAUSE_FAN];
```

Comment:
temporary saving of the current fanspeed , because , we will turn on and off in pause process , and restore when go out of pause